### PR TITLE
{chem}[intel/2015a] PLUMED 2.1.3 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.1.3-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://github.com/plumed/plumed2/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = ['PLUMED-2.0.4_eb-env-vars.patch']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.1.3-intel-2015a.eb]: patch missing

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a476a7136b3f17f39c2f for a full test report.

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.1.3-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.1.3-intel-2015a.eb]: no, the patch is already there, and still applies correctly

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c824bace452c952b60aa for a full test report.

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c69565bdfe5663e6ee5d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9e8a46e695f8a60afdf4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**